src/background/account/Account.ts

```diff
@@ -46,27 +61,29 @@ export class Account extends EventEmitter<AccountEvents> {
   isPendingNewUser: boolean;
 
   private static async writeCurrentUser(user: User) {
-    await browserStorage.set(currentUserKey, user);
+    await BrowserStorage.set(currentUserKey, user);
   }
 
   private static async writeCredentials(credentials: Credentials) {
-    await browserStorage.set(credentialsKey, credentials);
+    await SessionStorage.set(credentialsKey, credentials);
+    await LoginActivity.recordLogin();
   }
 
   private static async readCredentials() {
-    return await browserStorage.get<Credentials>(credentialsKey);
+    return await SessionStorage.get<Credentials>(credentialsKey);
   }
 
   private static async removeCredentials() {
-    return await browserStorage.remove(credentialsKey);
+    await SessionStorage.remove(credentialsKey);
+    await LoginActivity.recordLogout();
   }
 
   static async readCurrentUser() {
```

Comment:
Let's move these 2 functions to the top to place it near the `writeCurrentUser`?